group/assignment5/Assignment5.java

```diff
@@ -0,0 +1,149 @@
+package assignment5;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class Assignment5 {
+
+    private Graph graphOfLetters = new Graph();
+
+    public LinkedList<LinkedList<Character>> findAllAphabetsOfLanguage(List<String> dictionaryList) {
+        LinkedList<LinkedList<Character>> allPossibleAlphabets = new LinkedList<>();

```

Comment:
Is there a particular reason why you don't write the type when you instantiate objects? I usually do it as it looks safer.

Reply:
There's no particular reason, other then I personally find it easier (and cleaner) to do it this way. I don't think it's any less safe, because the compiler can infer the type arguments from the context (because the declaration is in the same line so you can easily see what the type is). When it comes down to it, I think this is just the way they thought us at school.